conversational-client/src/app/app.component.html

```diff
@@ -12,10 +10,10 @@
     </mat-toolbar>
 
     <!-- Custom UI for specific modes -->
-    <div *ngIf="currentMode === 'history_tutor'" class="prompt-mode-container">
+    <div *ngIf="agentId === 'history_tutor'" class="prompt-mode-container">
```

Comment:
Not necessarily for this PR unless it's convenient, but in general we should get rid of these its so adding a new agent can be encapsulated. I also think some agents (e.g. history tutor) don't need the chat history so either that's a behaviour in the agent component to switch it on or off, or maybe better the chat history is a component that each agent explicitly adds or doesn't add.

(I'll try and write this up for discussion)